docs/reST/ref/key.rst

```diff
@@ -10,27 +10,34 @@
 
 This module contains functions for dealing with the keyboard.
 
-The event queue gets ``pygame.KEYDOWN`` and ``pygame.KEYUP`` events when the
-keyboard buttons are pressed and released. Both events have a key attribute
-that is a integer ID representing every key on the keyboard.
-
-The ``pygame.KEYDOWN`` event has additional attributes ``unicode`` and
-``scancode``. ``unicode`` represents a single character string that is the fully
-translated character entered. This takes into account the shift and composition
-keys. ``scancode`` represents the platform-specific key code. This could be
-different from keyboard to keyboard, but is useful for key selection of weird
-keys like the multimedia keys.
-
-.. versionadded:: 2.0
+The :mod:`pygame.event` queue gets ``pygame.KEYDOWN`` and ``pygame.KEYUP``
+events when the keyboard buttons are pressed and released. Both events have
+``key`` and ``mod`` attributes.
+
+   * ``key``: an integer ID representing every key on the keyboard
+   * ``mod``: a bitmask of all the modifier keys that were in a pressed state
```

Comment:
Maybe a link to `_key-modifiers-label` at the end of this `mod` line?

Reply:
Submitted PR #1252 to address this.